Esp8266/fototimer/src/main.cpp

```diff
@@ -0,0 +1,284 @@
+                                        text-align: center; \
+                                    } \
+                                     </style> \
+                                    <title> FotoCelula1 </title>   \
+                                    </head> \
+                                    <body> \
+                                    <h1>Modo de Uso</h1> \
+                                    <div> \
+                                    <div style='width: 500px;'> \
+                                        <form action='/start_finish'> \
+                                            <button type='submit'>Inicio/Fim</button> \
+                                        </form> \
+                                        <form action='/round_course'> \
+                                            <button type='submit'>Circuito Fechado</button> \
+                                        </form> \
+                                    </div> \
+                                    </div> \
+                                    <br><hr><br> \
+                                    <div> \
+                                        <form action='/post' method='post'> \
+                                            <label for='numberInput' style='font-size: xx-large'>Digite ID: </label> \
+                                            <input type='number' id='message' height= 20px name='message' onchange='submitForm()'>                                        \
+                                        </form> \
+                                    </div> \
+                                    <br><hr><br> \
+                                    <div> \
+                                        <table> \
+                                            <tr> \
+                                                <th>ID</th> \
+                                                <th>Time</th> \
+                                            </tr>"
+                                                + text +
+                                        "</table> \
+                                    </div> \
+                                    <br><hr><br> \
+                                    <div> \
+                                        <form action='/delete'> \
+                                            <button type='submit'>Limpar Resultados</button> \
+                                        </form> \
+                                        <form action='/download'> \
+                                            <button type='submit'>Salvar Dados</button> \
+                                        </form> \
+                                    </div> \
+                                   </body> \
+                                  ");
+}
+
+void recalibrar() {
+  sensorValue = 0;
+  for (int i = 0; i < 10; i++) {
+    sensorValue = sensorValue + analogRead(sensorPin);
+    delay(200);
+  }
+  sensorValue = sensorValue / 10 - 50;
+}
```

Comment:
```suggestion
void recalibrar() {
  sensorValue = 0;
  sensorValue = analogRead(sensorPin);
  sensorValue = sensorValue - 50;
}
```
Removing delay and taking just one reading to ensure better performance.